bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/internal/module/handler/GenericCronTriggerHandler.java

```diff
@@ -79,4 +83,9 @@ public void run() {
             logger.debug("Tried to trigger, but callback isn't available!");
         }
     }
+
+    @Override
+    public @NonNull CronAdjuster getTemporalAdjuster() {
```

Comment:
Is this annotation correct? I mean the new interface uses `@Nullable`.

Reply:
Yes you're right, in TimeBasedTriggerHandler i've also used `@Nullable` but this isn't needed at all, so i've removed it at all places.